com.unity.formats.alembic/Runtime/Scripts/Importer/Behaviours/AlembicCustomData.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+
+namespace UnityEngine.Formats.Alembic.Importer
+{
+    /// <summary>
+    /// Class that stores ancillary data for the current Alembic Node.
+    /// </summary>
+    public class AlembicCustomData : MonoBehaviour
+    {
+        [SerializeField]
+        List<string> faceSetNames;
+
+        /// <summary>
+        /// Retrieves the set of Face Set names stored in the current Alembic node.
```

Comment:
I understand this is a set, but as we unfortunately have the same term in Face Set, it makes the sentence a bit unclear.
I would go direct with "Retrieves the Face Set names..."